lib/beaker-pe/install/pe_utils.rb

```diff
@@ -101,6 +101,30 @@ def install_ca_cert_on(host, opts)
           end
         end
 
+        # Return agent nodes with 'lb_connect' role that are not loadbalancers
+        def loadbalancer_connecting_agents
+          lb_connect_nodes = select_hosts(roles: ['lb_connect'])
+          lb_connect_agents = lb_connect_nodes.reject { |h| h['roles'].include?('loadbalancer')}
+        end
+
+        # Returns true if loadbalncer exists and is configured with 'lb_connect' role
+        def lb_connect_loadbalancer_exists?
+          if any_hosts_as?('loadbalancer')
+            lb_node = select_hosts(roles: ['loadbalancer'])
+            lb_node.first['roles'].include?('lb_connect')
+          end
+        end
+
+        #Returns loadbalancer if host is an agent and loadbalancer has lb_connect role
+        #@param [Host] agent host with lb_connect role
+        def get_lb_downloadhost(host)
+          downloadhost = master
+          if( ! host['roles'].include?('loadbalancer') &&  lb_connect_loadbalancer_exists?)
```

Comment:
@shaigy Sorry for not making it clear - the style concerns applied to the entire PR I just commented on the first one.

Can you please go through and remove parens from if statements etc.

Reply:
Done! Hopefully I have covered everything this time.